lib/app/modules/uscore_v3.0.1/us_core_smokingstatus_sequence.rb

```diff
@@ -177,9 +179,9 @@ def validate_resource_item(resource, property, value)
         assert !@observation.nil?, 'Expected valid Observation resource to be present'
 
         patient_val = @instance.patient_id
-        code_val = get_value_for_search_param(resolve_element_from_path(@observation_ary, 'code'))
-        date_val = get_value_for_search_param(resolve_element_from_path(@observation_ary, 'effectiveDateTime'))
-        search_params = { 'patient': patient_val, 'code': code_val, 'date': date_val }
+        category_val = get_value_for_search_param(resolve_element_from_path(@observation_ary, 'category'))
+        status_val = get_value_for_search_param(resolve_element_from_path(@observation_ary, 'status'))
```

Comment:
Why not `category` and `status` rather than `category_val` and `status_val`?

Reply:
I think I did that because encounter has a search by class+patient and class is a protected keyword.